src/index.js

```diff
@@ -46,26 +46,33 @@ const displayMovie = async() => {
 
 displayMovie();
 
-const postNewComments = (movieID='movie1', userName='Orcun', userComment='Trying') => {
-  axios.post(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/comments`, {
-    item_id: movieID,
-    username: userName,
-    comment: userComment
-  })
-}
-
-const renderItemsComments = async () => {
-  const getCommentsFromAPI = await getComments('movie1');
-  getCommentsFromAPI.forEach((i) => {commentHTML += `<p>${i.creation_date} ${i.username}: ${i.comment}</p>`})
-  console.log(commentHTML);
+const postNewComments = async (movieID, userName, userComment) => {
+  await axios.post(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/comments`, {
+    "item_id": movieID,
+    "username": userName,
+    "comment": userComment
+  });
+  showComments(movieID);
 }
 
 const getComments = async (movieId) => {
   const comments = await axios.get(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/comments?item_id=${movieId}`)
   return comments.data;
 }
 
-const showComments = () => {
+const showComments = async (id) => {
+  console.log(id);
+  try{ getCommentsFromAPI = await getComments(id)}catch{
+    popUpHtml(id);
+    commentHTML = '<p>No Comments Yet</p>'
+    getCommentsFromAPI = [];
+  };
```

Comment:
This try and catch block doesn't take action when the API responds with a 404. Please fix that.